arlas-tests/src/test/java/io/arlas/server/tests/ogc/wfs/WFSServiceIT.java

```diff
@@ -265,7 +264,7 @@ public void testDescribeFeatureWithAvailableColumn() throws Exception {
                         new ImmutablePair<>("VERSION", "2.0.0"),
```

Comment:
Shouldn't we add a `handleUnavailableCollection` test as it is the case for the exploration endpoints tests ?
same for stac, raw and opensearch endpoints ?

Reply:
ok